save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/ExecutionStatistics.kt

```diff
@@ -0,0 +1,50 @@
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
+package org.cqfn.save.frontend.components.basic
+
+import org.cqfn.save.execution.ExecutionDto
+import org.cqfn.save.execution.ExecutionStatus
+import react.Props
+import react.dom.button
+import react.dom.div
+import react.fc
+
+/**
+ * [Props] for execution statistics component
+ */
+external interface ExecutionStatisticsProps : Props {
+    /**
+     * And instance of [ExecutionDto], which should be passed from parent component
+     */
+    var executionDto: ExecutionDto?
+}
+
+/**
+ * A component which displays statistics about an execution from its props
+ *
+ * @param classes HTML classes for the enclosing div
+ * @return a functional react component
+ */
+fun executionStatistics(classes: String = "") = fc<ExecutionStatisticsProps> { props ->
+    val totalTests = props.executionDto?.run {
+        passedTests + failedTests + skippedTests
+    } ?: 0
+    val isInProgress = props.executionDto?.run { status == ExecutionStatus.RUNNING || status == ExecutionStatus.PENDING } ?: true
```

Comment:
May be `?: false` ?

Reply:
I thought that if `executionDto` is not available (i.e. we haven't etched if from backend yet) it makes more sense to use the same logic as for in-progress executions.